controllers/akodeploymentconfig/akodeploymentconfig_controller_avi_phase.go

```diff
@@ -239,14 +239,14 @@ func (r *AKODeploymentConfigReconciler) reconcileCloudUsableNetwork(
 	// Ensure network is added to the cloud's IPAM Profile as one of its
 	// usable Networks
 	var foundUsableNetwork bool
-	for _, net := range ipam.InternalProfile.UsableNetworkRefs {
-		if net == *(network.URL) {
+	for _, net := range ipam.InternalProfile.UsableNetworks {
```

Comment:
can you also change `UsableNetworkRefs` at `akodeploymentconfig_controller_intg_test.go:245` to fully deprecate it?

Reply:
good point, let me fix that too